dev/com.ibm.ws.security.wim.adapter.ldap/src/com/ibm/ws/security/wim/adapter/ldap/LdapAdapter.java

```diff
@@ -534,27 +534,32 @@ public Root login(Root root) throws WIMException {
             }
 
             String searchExpr = "@xsi:type=" + quote + qName + quote + " and "
-                                + SchemaConstants.PROP_PRINCIPAL_NAME + "=" + quote + principalName + quote;
+                            + SchemaConstants.PROP_PRINCIPAL_NAME + "=" + quote + principalName + quote;
 
             loginCtrl.setExpression(searchExpr);
             srchCtrl = getLdapSearchControl(loginCtrl, false, false);
             String[] searchBases = srchCtrl.getBases();
-            String sFilter = srchCtrl.getFilter();
+            String sFilter = null;
 
-            if (iLdapConfigMgr.getUseEncodingInSearchExpression() != null)
-                sFilter = LdapHelper.encodeAttribute(sFilter, iLdapConfigMgr.getUseEncodingInSearchExpression());
-
-            //Check is input principalName is DN, if it is DN use the same for login otherwise use userFilter
-            String principalNameDN = null;
-            try {
-                principalNameDN = new LdapName(principalName).toString();
-            } catch (InvalidNameException e) {
-                e.getMessage();
-            }
-            Filter userFilter = iLdapConfigMgr.getUserFilter();
-            if (principalNameDN == null && userFilter != null) {
-                sFilter = userFilter.prepare(principalName);
-                sFilter = setAttributeNamesInFilter(sFilter, SchemaConstants.DO_PERSON_ACCOUNT);
+            if (!iLdapConfigMgr.loginPropertyDefined()) {
+                //Check is input principalName is DN, if it is DN use the same for login otherwise use userFilter
+                String principalNameDN = null;
+                try {
+                    principalNameDN = new LdapName(principalName).toString();
+                } catch (InvalidNameException e) {
+                    e.getMessage();
```

Comment:
Ummmm.. are we just ignoring?

Reply:
Yes, I didn't change that behavior, but I can remove the e.getMessage.